src/helpers.ts

```diff
@@ -12,3 +14,22 @@ export const platform = selectPlatform({
   linux: 'linux',
   windows: 'windows'
 })
+
+/**
+ * Create a tray icon.
+ *
+ * @param unread True to create the unead icon; false to create the normal icon.
+ */
+export function createTrayIcon(unread: boolean): nativeImage {
```

Comment:
You probably want to use the interface here for the return type.

```suggestion
export function createTrayIcon(unread: boolean): NativeImage {
```

Reply:
OK. The confusing part is that `nativeImage` is a type alias or something of `NativeImage`, but I agree that using `NativeImage` seems to be the best here.